staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -135,7 +135,7 @@ func (f *SimpleRestOptionsFactory) GetRESTOptions(resource schema.GroupResource)
 		Decorator:               generic.UndecoratedStorage,
 		EnableGarbageCollection: f.Options.EnableGarbageCollection,
 		DeleteCollectionWorkers: f.Options.DeleteCollectionWorkers,
-		ResourcePrefix:          f.Options.StorageConfig.Prefix + "/" + resource.Group + "/" + resource.Resource,
```

Comment:
I understand from the linked issue discussion that we actually had a double-prefix. Do we have a test that we now have a single-prefix? Then I could agree.